includes/class-amp-story-post-type.php

```diff
@@ -38,6 +38,20 @@ class AMP_Story_Post_Type {
 	 */
 	const STORY_SQUARE_IMAGE_SIZE = 'amp-story-poster-square';
 
+	/**
+	 * The large dimension of the AMP Story poster images.
+	 *
+	 * @var int
+	 */
+	const STORY_LARGE_IMAGE_DIMENSION = 928;
```

Comment:
This could be overkill, but it seemed more DRY to avoid having `928` throughout this file.